homeassistant/components/vitrea/strings.json

```
@@ -0,0 +1,67 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "Failed to connect to the Vitrea device. Please check the host and port.",
+      "invalid_host": "Please enter a valid host address.",
+      "invalid_port": "Please enter a valid port number.",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "Host",
+          "port": "Port"
+        },
+        "data_description": {
+          "host": "IP address or hostname of the Vitrea device",
+          "port": "Port number"
+        },
+        "description": "Enter the connection details for your Vitrea device.",
+        "title": "Connect to Vitrea"
+      }
+    }
+  },
+  "entity": {
+    "cover": {
+      "blind": {
+        "name": "Blind"
+      }
+    },
+    "number": {
+      "timer_minutes": {
+        "name": "Timer"
+      }
+    },
+    "switch": {
+      "switch_with_timer": {
+        "name": "Switch with timer (boiler)"
+      }
+    }
+  },
+  "issues": {
+    "outdated_firmware": {
+      "description": "Your Vitrea device firmware version {current_version} is below the minimum required version {min_version}. To fix this issue:\n1. Open the Vitrea mobile app.\n1. Navigate to device settings.\n1. Select 'Update Firmware'.\n1. Wait for the update to complete.\n1. Restart Home Assistant.",
+      "title": "Device firmware is outdated"
+    }
+  },
+  "services": {
+    "set_timer": {
+      "name": "Set timer",
+      "description": "Set the timer value for a switch with timer functionality.",
```

Action descriptions should use third person singular to sound descriptive:

```suggestion
      "description": "Sets the timer value for a switch with timer functionality.",
```